docs/blog/2019-11-19-gatsby-brings-speed-to-mediacurrent-com/index.md

```diff
@@ -0,0 +1,70 @@
+- We found short codes for files from our Drupal 7 instance that were processed in the Jekyll build process.
+  - We band-aided this fix with our Jekyll module. We had to go back and re-publish this information correctly.
+- Audio file information.
+  - This was patched through our Jekyll module, but we added a JSON:API field enhancer which added the necessary data to show audio files.
+- External JS calls. (Acquia Lift, Pardot)
+  - Since Gatsby is ultimately a React application, the way to insert tracking cookies, and other 3rd Party scripts required workarounds. Fortunately, the gatsby-ssr API made this easy. We used the onRenderBody lifecycle to confirm which environment we were on, and inserted the required script tags accordingly.
+
+## Final Thoughts
+
+Ultimately, the decision to convert Mediacurrent.com to a Gatsby front-end was a wise move. Considering the issues we inherited by using a static-site generator that wasn’t intentionally built as a CMS front-end, Gatsby has been a welcome improvement. The benefits we’re experiencing with Gatsby are:
+
+- No wait time to apply Drupal module updates. The Gatsby front-end and Drupal back-end are completely decoupled.
+- Simplified decoupled architecture with no middle-men.
+- Front-end design updates can be deployed without concern for impact on back-end logic.
+- [Gatsby Preview](https://www.gatsbyjs.com/preview/) allows us to view content updates in real-time prior to publishing to production.
+- Build times are now much improved using Gatsby’s new Build feature (in beta). We are seeing builds come in at 4-8 minutes on average compared to Jekyll which took up to 15 minutes to compile. We also know that incremental builds are coming soon and will provide substantial improvements.
+
+To learn more about Mediacurrent’s work with Gatsby, check out Ben Robertson’s [Gatsby Accessibility Starter](/starters/benjamingrobertson/gatsby-starter-accessibility/) and these additional resources:
+
+- [Digital Display With Decoupled Drupal 8 & Gatsby](/blog/2019-07-09-digital-display-drupal-gatsby/) - City of Sandy Springs Case Study
+- [Move Fast, Don’t Break Things: Trends in Modern Web Dev with Mediacurrent](https://youtu.be/QiocnDGnKfs) - Gatsby Days presentation by Ben Robertson
+- [Maximizing the Potential of Component-Driven Drupal Development with Gatsby](/blog/2019-06-28-component-driven-drupal-development/) - Blog post
+- [Gatsby for Agencies and Teams: Better, Faster, Smarter](/gatsby-for-agencies/?_ga=2.75802454.258677743.1570125231-898453189.1565876794) - Webinar
```

Comment:
- webinar is on the `.com` site (there is also an https://www.gatsbyjs.org/docs/gatsby-for-agencies on `.org`)
- `ga` code removed (or is this inteded?)
```suggestion
- [Gatsby for Agencies and Teams: Better, Faster, Smarter](https://www.gatsbyjs.com/gatsby-for-agencies/) - Webinar
```